src/SablierV2MerkleLockupFactory.sol

```diff
@@ -63,26 +63,26 @@ contract SablierV2MerkleLockupFactory is ISablierV2MerkleLockupFactory {
         ISablierV2LockupTranched lockupTranched,
         MerkleLockupLT.TrancheWithPercentage[] memory tranchesWithPercentages,
         uint256 aggregateAmount,
-        uint256 recipientsCount
+        uint256 recipientCount
     )
         external
         returns (ISablierV2MerkleLockupLT merkleLockupLT)
     {
         // Calculate the sum of percentages and durations across all tranches.
-        UD60x18 totalPercentage;
+        uint64 totalPercentage;
         uint256 totalDuration;
         for (uint256 i = 0; i < tranchesWithPercentages.length; ++i) {
-            UD60x18 percentage = (tranchesWithPercentages[i].unlockPercentage).intoUD60x18();
-            totalPercentage = totalPercentage.add(percentage);
+            uint64 percentage = tranchesWithPercentages[i].unlockPercentage.unwrap();
+            totalPercentage = totalPercentage + percentage;
             unchecked {
                 // Safe to use `unchecked` because its only used in the event.
                 totalDuration += tranchesWithPercentages[i].duration;
             }
         }
 
-        // Checks: the sum of percentages equals 100%.
-        if (!totalPercentage.eq(ud(1e18))) {
-            revert Errors.SablierV2MerkleLockupFactory_TotalPercentageNotEqualOneHundred(totalPercentage.intoUint256());
+        // Check: the sum of percentages equals 100%.
+        if (totalPercentage != uUNIT) {
```

Comment:
Actually, the reader should want to understand what `uUNIT` is. It's a sign that 1e18 holds a special value in this context, namely, that it stands for 100%.

Reply:
I agree with Andrei that using `1e18` here is more developer-friendly as they wouldn't have to lookup at the value of `uUNIT`. But if we want to give `1e18` special treatment because of the percentages, wouldn't it be more appropriate to rename it to `100_PERCENT` than `uUINT`?

Reply:
Creating a new variable would increase the gas cost.

I find the comment explanatory enough. Percentage calculations are a common vector of attacks in smart contracts, so I prefer specificity (custom constant `uUNIT` in this case) over readability (1e18). The latter provides 'lower resolution' because the user has to hope that 1e18 is indeed in accord with the percentage system used by PRBMath.

Reply:
Fair enough. Lets go with `uUINT`.